### DISCUSSION TITLE
@emotion peer dependency

### DISCUSSION THREAD

**Discussion by abadire** [mantinedev/mantine#2017] [Q&A] [answered]
Hello! Regarding [this issue](https://github.com/mantinedev/mantine/issues/2012). Why is @emotion/react is peer dependency for @mantine/style and should be installed by the user? If a project with Mantine won't start without it, then why is it not a regular dependency? Shouldn't peer deps be optional? Just curious about the rationale behind it.

**Comment by rtivital** [comment, marked as answer]
See https://github.com/mantinedev/mantine/issues/1715

**Reply by abadire** [reply]
Thanks!
